app/database/migrations/2015_05_11_201708_create_users_table.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->string('password', 200);
             $table->rememberToken();
             $table->string('items', 200)->nullable();
-            $table->integer('money')->default(16);
+            $table->decimal('money',6,2)->default(16.00);
```

Comment:
We've already talked about how the money is going to be an integer because we're not going to be dealing with non-whole values